### PR TITLE
Add disk monitoring integration

### PR DESCRIPTION
**Pull request by calebhailey**
TODO: 

- [x] README.md
- [x] sensu-resources.yaml
- [x] sensu-integration.yaml
- [x] logo.png

### CODE REVIEW

**Comment by calebhailey** [comment]
Just need to add a logo then this one is ready to go.

**Comment by calebhailey** [comment]
I think we can skip the logos for now. In fact, I think it might be advantageous to add them en masse just before we launch.

File: integrations/system/disk-monitoring/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+<!-- Sensu Integration description; supports markdown -->
+
+The disk monitoring integration integration provides disk usage monitoring for Linux, Windows, and macOS system.
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration provides the following resources:
+
+* "disk-usage" [check]
+* "sensu/check-disk-usage:0.6.0" [asset]
```
Comment (jspaleta) [inline integrations/system/disk-monitoring/README.md:12]: Small thing..
Should be we adding the versioned asset name here as best practice?  I think so far we've been inconsistent with this in the first batch of integrations.

File: integrations/system/disk-monitoring/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+1. Optionally set the `disk_usage_warning_threshold` agent [annotation] to override the default value (`80.0`%).
+
+1. Optionally set the `disk_usage_critical_threshold` agent [annotation] to override the default value (`90.0`%).
+
+## Plugins
+
+<!-- Links to any Sensu Integration dependencies (i.e. Sensu Plugins) -->
+
+This integration does not require any [Plugins].
```
Comment (jspaleta) [inline integrations/system/disk-monitoring/README.md:41]: This is incorrect,
need to reference the github and bonsai listings for the check plugin here.

**Review by jspaleta — CHANGES_REQUESTED** [review]
Update Plugins section of readme

File: integrations/system/disk-monitoring/README.md (hunk 1 above)
Comment (calebhailey) [reply integrations/system/disk-monitoring/README.md:12]: I don't think this is a deal breaker one way or the other, but let's go with versioned references for now.

**Review by jspaleta — APPROVED** [review]

**Pull request merged**